payloads/external/iPXE/Kconfig

```diff
@@ -63,6 +63,13 @@ config PXE_ROM_FILE
 	help
 	  The path and filename of the file to use as PXE ROM.
 
+config PXE_SERIAL_CONSOLE
```

Comment:
@miczyg1 looks like it is, are we sure we can justify that ? Maybe you should send patch for review in community ?

Reply:
It is a good idea, but I wanted to wait till SeaBIOS 1.11.0 will be in official coreboot. It turns out that it has been merged few hours ago. At the time I created this PR, it was not merged. We can treat it as a temporary solution, which is a must for now (we are running short on time with release)